altair/vegalite/v5/schema/_config.py

```diff
@@ -93,7 +93,7 @@
 
 class AreaConfigKwds(TypedDict, total=False):
     """
-    :class:`AreaConfig` ``TypedDict`` wrapper.
+    :class:`altair.AreaConfig` ``TypedDict`` wrapper.
```

Comment:
Previously these didn't work as links in the API reference.

Now they do

![image](https://github.com/user-attachments/assets/46985e0b-0a2a-418c-a323-52a702131b41)